src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -2,18 +2,8 @@
 
 public class BooleanExpression {
 
-    /**
-     * <p>Реализуйте метод booleanExpression(boolean a, boolean b, boolean c, boolean d),
-     * возвращающий true, если среди четырех его аргументов ровно два истинны (любые)
-     * во всех остальных случаях метод должен возвращать false.</p>
-     *
-     * <p>Вам надо только проанализировать переданные в метод booleanExpression
-     * значения (a, b, c, d) и вернуть результат.</p>
-     *
-     * <p>Это задание необходимо решать с использованием логических операторов,
-     * без использования конструкций if-else</p>
-     */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
+        return a != b & c != d | a == b & c == d & a != c;
```

Comment:
Why have decided to use here bitwise operators | and & instead of logical || and && ?

Reply:
changed to logical (cause it worked))

Reply:
ok, but it's important to understand the difference, hope you've read about it🙂